portal/templates/utilisation-report-service/utilisation-report-upload/utilisation-report-upload.njk

```diff
@@ -19,52 +19,58 @@
         }) }}
     {% endif %}
     <h1 class="govuk-heading-l" data-cy="main-heading">
-        {% if dueReportDates.length > 0 %}
+        {% if dueReportPeriods.length > 0 %}
             Report GEF utilisation and fees
         {% else %}
             Report not currently due for upload
         {% endif %}
     </h1>
-    {% if dueReportDates.length === 0%}
+    {% if dueReportPeriods.length === 0 %}
         <p class="govuk-body" data-cy="next-due-report-text">
             The {{ nextReportPeriod }} report can be uploaded from {{ nextReportPeriodSubmissionStart }}.
         </p>
+        {% if lastUploadedReportPeriod === undefined %}
+        <p class="govuk-body" data-cy="no-reports">
+            No reports have been submitted.
+        </p>
```

Comment:
I think I added this to handle when no reports had been submitted and none were due but if that throws `problem with service` we probably don't need this anymore